<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoReviewForm</class>
 <widget class="QWidget" name="PhotoReviewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QLabel" name="photo_label">
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="yes_pushButton">
       <property name="text">
        <string>Да</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="no_pushButton">
       <property name="text">
        <string>Нет</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
